include/powsybl/iidm/OperationalLimitsHolder.hpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_OPERATIONALLIMITSHOLDER_HPP
+#define POWSYBL_IIDM_OPERATIONALLIMITSHOLDER_HPP
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include <powsybl/iidm/LimitType.hpp>
+#include <powsybl/iidm/OperationalLimits.hpp>
+#include <powsybl/iidm/Validable.hpp>
+#include <powsybl/stdcxx/range.hpp>
+#include <powsybl/stdcxx/reference.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+class ActivePowerLimitsAdder;
+class ApparentPowerLimitsAdder;
+class CurrentLimitsAdder;
+class FlowsLimitsHolder;
+class Identifiable;
+
+class OperationalLimitsHolder : public Validable {
+public:  // Validable
+    std::string getMessageHeader() const override;
+
+public:
+    OperationalLimitsHolder(Identifiable& identifiable, std::string&& attributeName);
+
+    explicit OperationalLimitsHolder(std::string&& attributeName);
+
+    OperationalLimitsHolder(const OperationalLimitsHolder&) = default;
+
+    // NOLINTNEXTLINE(performance-noexcept-move-constructor): move constructor of std::map is not marked noexcept
+    OperationalLimitsHolder(OperationalLimitsHolder&&) = default;  // NOSONAR
+
+    ~OperationalLimitsHolder() noexcept override = default;
+
+    OperationalLimitsHolder& operator=(const OperationalLimitsHolder&) = default;
+
+    OperationalLimitsHolder& operator=(OperationalLimitsHolder&&) = default;
```

Comment:
I'm not totally sure we should make move constructor and assignment operator public because it could break the backward link (m_identifiable). What do you think?